### PR TITLE
Fix ResourceTiles on mobile and add basic resize slider for tiles

### LINKED ISSUE #8367: [Tiles view] Fix mobile display of tiles grid
### Expected behaviour
Tiles should fill 100% of the display width on mobile display

### Actual behaviour
One tile gets displayed, but doesn't fill the full width

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Description
Didn't invest the time to create a `OcSlider` component but kept it inline, also doesn't fetch bigger tiles thumbnails since the current resolution seems quite reasonable even on bigger tiles.

## Related Issue
- Fixes #8367
- Addresses parts of #8368

### CODE REVIEW

File: packages/design-system/src/tokens/ods/size.yaml [hunk 1]
```diff
@@ -22,3 +22,8 @@ size:
     value: 22px
   form-check-default:
     value: 14px
+  tiles:
+    resizeable:
```
Comment (kulmann) [inline packages/design-system/src/tokens/ods/size.yaml:26]:
```suggestion
    resize-step:
```
or something similar? resizable (typo btw) sounds like a boolean.

**Review by kulmann — CHANGES_REQUESTED** [review]

File: packages/design-system/src/tokens/ods/size.yaml (hunk 1 above)
Comment (pascalwengerter) [reply packages/design-system/src/tokens/ods/size.yaml]: Great idea, and thanks for the educate - turns out past me already made this mistake :D https://github.com/owncloud/web/pull/5052

File: packages/design-system/src/tokens/ods/size.yaml [hunk 2]
```diff
@@ -22,3 +22,8 @@ size:
     value: 22px
   form-check-default:
     value: 14px
+  tiles:
+    resize-step:
+      value: 12rem
+    default:
```
Comment (kulmann) [inline packages/design-system/src/tokens/ods/size.yaml:28]: The default is not in use. Delete it?

File: packages/design-system/src/tokens/ods/size.yaml [hunk 3]
```diff
@@ -22,3 +22,8 @@ size:
     value: 22px
   form-check-default:
     value: 14px
+  tiles:
+    resize-step:
```
Comment (kulmann) [inline packages/design-system/src/tokens/ods/size.yaml:26]: If `resize-step` is the only size value for tiles, we could omit the nesting an just have something like
```yml
  tiles-step:
    value: 12rem
```

File: packages/design-system/src/tokens/ods/size.yaml (hunk 2 above)
Comment (pascalwengerter) [reply packages/design-system/src/tokens/ods/size.yaml:28]: Good point, thx. It's meant to be used for non-resizable tiles (like the ones on the spaces-projects view), I added the token without actually using it in the component 🙈 see latest commit in this PR

Comment (kulmann) [reply packages/design-system/src/tokens/ods/size.yaml:28]: aaaah, now it makes sense 😅

**Review by kulmann — APPROVED** [review]

**Pull request merged**